components/Editor/index.tsx

```diff
@@ -284,6 +287,7 @@ const Editor = ({ readOnly = false }: Props) => {
 
   const handleCodeTypeChange = (option: OnChangeValue<any, any>) => {
     const { value } = option
+    const lastCodeType = codeType
```

Comment:
do you really need to redefine codeType?

Reply:
I remove the ```codeType``` and the statement that checks whether ``` lastCodeType``` is equal to bytecode or mnemonic in the if statement.